### PR TITLE
convert media related Android.mk to Android.bp

### PR DESCRIPTION
**Pull request by zhangyichix**
Since Android.mk is legacy, we start to use
Android.bp instead of Android.mk.

Tracked-On: OAM-111850

### CODE REVIEW

**Comment by zhangyichix** [comment]
@TianmiChen @Shao-Feng @YuanjunHuang please help to take a review, thanks!

File: Android.bp [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        ".",
+        "va"
+    ],
+
+    header_libs: [
+        "libutils_headers",
+    ],
+
+    include_dirs: [
+        "vendor/intel/mediasdk_c2/c2_utils/include",
```
Comment (TianmiChen) [inline Android.bp:41]: Please remove these include folders.
Libva is an independent module and should not have correlation with os and hw platform.

Comment (zhangyichix) [reply Android.bp:41]: sorry, I just copy it from there, forgot to remove it.😂

File: Android.bp [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    name: "libva_headers",
+
+    export_include_dirs: [
+        ".",
+        "va",
+        "va/drm"
+    ],
+
+    vendor: true,
+}
```
Comment (TianmiChen) [inline Android.bp:31]: I see there is a statement LOCAL_COPY_HEADERS_TO := libva/va in original Android.mk, which is used to copy va related headers to out/target/product/caas/obj/include/libva/va. Do we need to consider to copy the headers as well in Android.bp? Not sure whether media driver can be built well if we don't copy the va headers. 

Thanks,
Tianmi

Comment (zhangyichix) [reply Android.bp:31]: LOCAL_COPY_HEADERS_TO is unsupported in soong, I've modified the android.mk in libmfx-gen, will commit the pr later.

**Review by TianmiChen — APPROVED** [review]

**Pull request merged**